README.md

```diff
@@ -59,5 +59,8 @@ For more advanced configuration, please refer to the `curl` documentation.
   * `git tag --message 'Version 1.2.3' 1.2.3`
   * `git push --tags`
 * Create a release on GitHub for the new tag.
-* Run `./make_release.bash 1.2.3 .`
-* Upload `cs-api-1.2.3.tar.gz` to the GitHub release.
+* Get the binaries from the [«Actions» tab] on GitHub
```

Comment:
**Suggestion:** Provide more detailed instructions

I went to "Actions" tab and could not find where to get any binary from there. Do you think you could provide more detailed instructions?

(+ nitpick: we should use English quotes `"` instead of French quotes `« ... »`)

Reply:
How about „those“ ^^? Or like “this” XD! Anyway, updated. And I clarified the instructions somewhat.